l2geth/cmd/utils/flags.go

```diff
@@ -1176,11 +1171,20 @@ func setRollup(ctx *cli.Context, cfg *rollup.Config) {
 	if ctx.GlobalIsSet(SequencerClientHttpFlag.Name) {
 		cfg.SequencerClientHttp = ctx.GlobalString(SequencerClientHttpFlag.Name)
 	}
-	if ctx.GlobalIsSet(SchedulerAddressFlag.Name) {
-		cfg.SchedulerAddress = common.HexToAddress(ctx.GlobalString(SchedulerAddressFlag.Name))
-	}
-	if ctx.GlobalIsSet(SequencerModeFlag.Name) {
-		cfg.SequencerMode = ctx.GlobalBool(SequencerModeFlag.Name)
+	if ctx.GlobalIsSet(RollupRoleFlag.Name) {
+		str := ctx.GlobalString(RollupRoleFlag.Name)
+		switch str {
+		case "scheduler", "SCHEDULER":
+			cfg.RollupRole = rollup.SCHEDULER_NODE
+		case "sequencer", "SEQUENCER":
+			cfg.RollupRole = rollup.SEQUENCER_NODE
+		case "verifier", "VERIFIER":
+			cfg.RollupRole = rollup.VERIFIER_NODE
+		case "replica", "REPLICA":
+			cfg.RollupRole = rollup.SEQUENCER_NODE
```

Comment:
@Kukoomomo @curryxbo 
Does a `replica` node have any different behavior with `sequencer` node?

Reply:
The current behavior is similar to that of verifier, but the data source is different. sequencer can be used to replace replica later